rpc/rest/security/model/http-security/httpsecurity.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+// Package http_security provides basic definition of user and permission group
+package http_security;
+
+message User {
+    string name = 1;
+    string password_hash = 2;
+    repeated string permissions = 3;            // Permission name, user is allowed for more permission groups
+}
+
+message PermissionGroup {
+    string name = 1;
+    message Permissions {
+        string url = 1;                         // URL enabled for the permission
```

Comment:
This might also need to be little more generic instead of specifics like: *URL* or *method* so it can more easily fit other types, like the GRPC. Try to think about some alternatives that would fit both.

Reply:
If we want to use a single model to define permission groups for all RPC types, I think it would be better to define a separate set of permission field for each of them, instead of thinking about some generic fields which may become confusing.